wondrous-app/components/OnboardingDao/AddImages/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './component';
```

Comment:
if there's no other files in the folder can we simply transfer the component.ts here and export default the AddImages component ?